speaking-guidelines.md

```diff
@@ -0,0 +1,70 @@
+# Triangle DevOps – Guidelines for Speakers
+
+Thank you for your interest in presenting to Triangle DevOps. We very much
+appreciate it.
+
+We have a few guidelines we ask you to understand about presenting to our group.
+
+## Preparing your talk
+
+First, talks should aim for about 45 minutes to an hour of length. Lean for
+shorter than an hour if you're expecting your talk to generate questions.
+
+Please let us know how long you're expecting your talk to run ahead of time so
+we can make sure time is budgeted accordingly.
+
+Second, we have a mix of experiences and backgrounds in our group. Be careful
+about assuming that the acronyms, terms, concepts and skills you'll be speaking
+about are universally understood. We want to hear from a broad range of opinion,
+experience and expertise. Help bring our members along with you by taking a
+moment to explain acronyms and briefly explain foundational concepts. Look over
+your material and ask what a newcomer to the field might not understand.
+
+Third, please review our [Code of Conduct][conduct] with respect to your talk.
+We want your talk to reflect the inclusiveness we seek for our community. To
+that end, we kindly ask as you prepare that you:
+
+* Use non-gendered, inclusive, language like "folks" over "guys"
+* Be mindful that animated GIFs, memes and memespeak may have a limited
+  audience. They're fine to use within the boundaries of the Code of Conduct,
+  but make sure you read the room to see if everyone's catching on. When in
+  doubt, explain the meaning behind the GIF, meme or memespeak.
```

Comment:
I'm terrified of the day someone decides to put sexualized images in their talk. I never expect it, but it's happen at conferences often enough it might be worth calling out as an example of what not to do.